spec/controllers/preferences_interview_controller_spec.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe PreferencesInterviewController do
+  let(:step) { assigns(:step) }
+  let(:invalid_params) { { step: { interview_preference: "telepathy" } } }
+  let(:step_class) { PreferencesInterview }
+
+  before { session[:snap_application_id] = current_app.id }
+
+  include_examples "step controller"
+
+  describe "#edit" do
+    it "assigns the attributes to the step" do
```

Comment:
this would be for a separate PR but I think it would be cool to get something like this into the shared example group